mods/bones/init.lua

```diff
@@ -16,6 +16,27 @@ local function is_owner(pos, name)
 	return false
 end
 
+local drop = function(pos, itemstack)
```

Comment:
```suggestion
local function drop(pos, itemstack)
```

Reply:
I cannot find anything about this in the guidelines. Furthermore this function had been there before. :)

Reply:
Well the function right above it uses the `local function` syntactic sugar. (I also prefer this style and it costs nothing to apply it consistently as we go.)